source/css/elements/_accordion.scss

```diff
@@ -0,0 +1,20 @@
+details {
+  overflow: hidden;
+  display: block;
+  width: 100%;
+}
+
+summary {
+  padding: 20px;
+  margin-bottom: 1px;
+  background-color: $background_offset_white;
+  cursor: pointer;
+  font-weight: 600;
+  color: $colour_base;
+  @include fontSize(18px, 2.125);
+  line-height: 2.125;
+}
```

Comment:
What is `$gutter`…?  I don't even know.  I _think_ `padding: 20px` is a bit much—I'd aim for `padding: 15px` personally here (suggested as `padding: $gutter;`.

1. Does `@include fontSize(18px, 2.125);` not include `line-height`?
2. The margin between is lacking.  Maybe my suggestion is too much/too little?  They should definitely feel "grouped" in my book.
<img width="789" alt="Screen Shot 2021-05-25 at 12 34 25 PM" src="https://user-images.githubusercontent.com/4203271/119422809-efeba600-bd55-11eb-80e6-335833b99551.png">

Also just keen if you get this same feedback from Sam Straun 😉 

```suggestion
summary {
  padding: $gutter;
  background-color: $background_offset_white;
  cursor: pointer;
  font-weight: 600;
  color: $colour_base;
  @include fontSize(18px, 2.125);
  line-height: 2.125;
  
  & + & {
    margin-top: $gutter/2;
  }
}
```

Reply:
Ha, `$gutter` is `20px`